addons/binding/org.openhab.binding.tadoac/cfg/tadoac.cfg

```diff
@@ -0,0 +1 @@
+# TadoAC Binding Default Configuration
```

Comment:
This has been generated - I also wonder whether we should keep it, see the discussion here: https://github.com/eclipse/smarthome/pull/3278#issuecomment-307767805